server.php

```diff
@@ -65,8 +65,8 @@ function valet_default_site_path($config)
 
 $siteName = basename(
     // Filter host to support wildcard dns feature
-    valet_support_wildcard_dns($_SERVER['HTTP_HOST']),
-    '.'.$valetConfig['tld']
+    valet_support_wildcard_dns(explode(':', $_SERVER['HTTP_HOST'])[0]),
+    '.'.$valetConfig['domain']
```

Comment:
Hi, `domain` here should be changed to `tld`

Reply:
Noted, thanks.